### DISCUSSION TITLE
Localization updates

### DISCUSSION THREAD

**Discussion by pattlebass** [pattlebass/Music-DJ#9] [Announcements]
# What's this
This page is for translators. Every time there are new strings that need to be translated I'll post here. Be sure to subscribe to this post!
Also, here's the [spreadsheet](https://docs.google.com/spreadsheets/d/1VEE4aROwNFXsZTRbMOShlv2E8XcYVBYYEVsRbFYKBog/edit?usp=sharing) you can use.

## How to contribute
Check [CONTRIBUTING.md](https://github.com/pattlebass/Music-DJ/blob/main/CONTRIBUTING.md)

If you see anything wrong with the current translation, please open an issue.

**Comment by pattlebass** [comment]
@StarryWindy I added one last key before 1.4-beta:
`SETTING_WEB_WARNING`: Some settings are not available in the web version.

Can you send me the translation here?

**Reply by StarryWindy** [reply]
`SETTING_WEB_WARNING`: Beberapa pengaturan tidak tersedia di versi web.

That should do it 🙂

**Comment by pattlebass** [comment]
@StarryWindy New keys for v1.4.1:
`DIALOG_PROGRESS_AFTER_EXPORT`: You can find the recording here: %s
Where `%s` is the path to the file

`DIALOG_PROGRESS_ERROR`: An error occured. Make sure you gave all the necessary permissions. Error code: %s.

**Reply by StarryWindy** [reply]
Alright, here goes:

`DIALOG_PROGRESS_AFTER_EXPORT`: Anda bisa menemukan hasil rekaman di sini: %s
`DIALOG_PROGRESS_ERROR`: Terjadi kesalahan. Pastikan anda memberikan semua izin yang diperlukan. Kode kesalahan: %s.

**Comment by pattlebass** [comment]
@StarryWindy New keys:
`DIALOG_LOAD_BTN_SHARE`: Share
`DIALOG_LOAD_BTN_DELETE`: Delete

![image](https://user-images.githubusercontent.com/49322676/196949448-7bc440b3-9ea0-442a-9b18-ca8eed1613c6.png)

**Reply by StarryWindy** [reply]
Alright, @pattlebass.
`DIALOG_LOAD_BTN_SHARE`: Bagikan
`DIALOG_LOAD_BTN_DELETE`: Hapus

**Reply by pattlebass** [reply]
@StarryWindy Sorry for bothering you but I added three more:

`DIALOG_CONFIRMATION_TITLE_NEW_PROJECT`: Create new project?
`DIALOG_CONFIRMATION_BODY_NEW_PROJECT`: Any unsaved changes will be lost.
`BTN_NEW_PROJECT`: New project

Thank you very much!

**Reply by StarryWindy** [reply]
Thanks, @pattlebass.
`DIALOG_CONFIRMATION_TITLE_NEW_PROJECT`: Buat proyek baru?
`DIALOG_CONFIRMATION_BODY_NEW_PROJECT`: Setiap perubahan yang belum disimpan akan hilang.
`BTN_NEW_PROJECT`: Proyek baru

**Comment by StarryWindy** [comment]
Hi @pattlebass, here's a new Indonesian translation line.
`DIALOG_LOAD_BTN_LINK`: Salin tautan

**Comment by pattlebass** [comment]
@StarryWindy Some new lines:
`BTN_COPY`: Copy
`BTN_PASTE`: Paste
`BTN_PLAY_COLUMN`: Play column

**Reply by StarryWindy** [reply]
Sorry to keep you waiting!

`BTN_COPY`: Salin
`BTN_PASTE`: Tempel
`BTN_PLAY_COLUMN`: Putar kolom

**Comment by pattlebass** [comment]
Hey @StarryWindy ! It's been a while. I am working on a new release and would appreciate your help translating a few new strings.


| keys | en |
|---|---|
| BTN_NEW_COLUMN | New column |
| BTN_DUPLICATE | Duplicate |
| BTN_DONATE | Donate |
| BTN_UNDO | Undo |
| BTN_REDO | Redo |
| DIALOG_LOAD_BTN_EXPORT | Export |
| TUTORIAL_TILE_QE_SWIPE | Swipe up or down on a tile to quick edit it. |
| TUTORIAL_TILE_QE_SCROLL | Scroll up on a tile (or press %s when focused) to quick edit it. |
| TUTORIAL_COLUMN_MOVE | Long press on a column (or press %s when focused) to move it. |

For context, now when you swipe or point your mouse and scroll on a tile it increments/decrements its sample.

**Reply by StarryWindy** [reply]
It's been a while, @pattlebass ! I'd like to know, by "when focused", does it mean when that specific tile or column lights up or something when hovered? And for a meantime, I'm bringing these lines:

`BTN_NEW_COLUMN`: Kolom baru
`BTN_DUPLICATE`: Duplikat
`BTN_DONATE`: Donasi
`BTN_UNDO`: Urungkan
`BTN_REDO`: Ulangi
`DIALOG_LOAD_BTN_EXPORT`: Ekspor
`TUTORIAL_TILE_QE_SWIPE`: Geser ke atas atau ke bawah di balok untuk mengeditnya dengan cepat.

**Reply by pattlebass** [reply]
Thanks!

> I'd like to know, by "when focused", does it mean when that specific tile or column lights up or something when hovered?

It's about the blue outline that appears when you use keyboard controls. I didn't know how to phrase it in as few words as possible.



https://github.com/user-attachments/assets/c623019b-4937-4671-8b82-7c09fe6fdfd4

**Reply by StarryWindy** [reply]
Oh, I see. In this case, ~~I think the word "selected" might be more fitting, probably~~ on second thought, the word "highlighted" might fit much better. Even though you can check on other similar words as well, just to be sure.

So for a meantime I'll go with this translation (please let me know if there are changes ahead, thank you 😄):
`TUTORIAL_TILE_QE_SCROLL`: Gulir ke atas di balok (atau tekan %s jika disorot) untuk mengeditnya dengan cepat.
`TUTORIAL_COLUMN_MOVE`: Tekan lama di kolom (atau tekan %s jika disorot) untuk memindahkan kolomnya.

**Reply by pattlebass** [reply]
One more:
`BTN_CUT` - Cut

**Reply by StarryWindy** [reply]
Alright!
`BTN_CUT`: Potong
